templates/Dockerfile.j2

```diff
@@ -23,10 +29,8 @@ RUN groupadd -g 1000 elasticsearch && adduser -u 1000 -g 1000 -d /usr/share/elas
 
 WORKDIR /usr/share/elasticsearch
 
-# Download/extract defined ES version. busybox tar can't strip leading dir.
+# Download and extract defined ES version. busybox tar can't strip leading dir.
 RUN wget --progress=bar:force {{ url_root }}/{{ tarball }} && \
-    EXPECTED_SHA=$(wget -O - {{ url_root }}/{{ tarball }}.sha1) && \
-    test $EXPECTED_SHA == $(sha1sum {{ tarball }} | awk '{print $1}') && \
     tar zxf {{ tarball }} && \
```

Comment:
With Alpine gone I can simplify this to extract by skipping the leading path; will address in another PR.